libraries/tock-register-interface/src/fields.rs

```diff
@@ -479,6 +480,112 @@ macro_rules! register_bitmasks {
     };
 }
 
+/// Helper macro for defining debugging of a field.
+#[macro_export]
+#[cfg(not(feature = "register_debug"))]
+macro_rules! impl_register_debug {
+    ($($t:tt)*) => {};
+}
+
+/// Helper macro for defining debugging of a field.
+#[macro_export]
+#[cfg(feature = "register_debug")]
+macro_rules! impl_register_debug {
```

Comment:
If not gated behind a feature, is there any reason not to merge this with the `register_bitmasks!` macro? I'm skittish to have two such complex macro rules which parse the exact same syntax.

Reply:
I have changed this, now it uses the parsing from `register_bitmask`, since we can't have features inside 1 macro, I just enabled `debug` module on all times.
The `debug` related traits and implementations should be optimized away when we don't use the `debug` function.